controller/linodemachine_controller.go

```diff
@@ -209,6 +209,9 @@ func (r *LinodeMachineReconciler) reconcile(
 		}
 	}()
 
+	// Add the finalizer if not already there
+	controllerutil.AddFinalizer(machineScope.LinodeMachine, infrav1.GroupVersion.String())
```

Comment:
since youre playing with this line could you add a patch like @eljohnson92 PR earlier to cluster https://github.com/linode/cluster-api-provider-linode/pull/64/files#diff-d6148d3b8d79d76ad02a5e26b74a8bbff6541058ae664f3c10e5dd97accffd76R99-R113

Reply:
Good call, I'll get that added 👍